package.json

```diff
@@ -40,7 +40,8 @@
     "proj4": "^2.4.3",
     "text-encoding-utf-8": "^1.0.1",
     "three": "^0.86.0",
-    "whatwg-fetch": "^2.0.2"
+    "whatwg-fetch": "^2.0.2",
+    "three.meshline": "^1.0.3"
```

Comment:
Also, you didn't update `package-lock.json` (that requires a recent enough version of npm)